contracts/mocks/VariableYieldMathWrapper.sol

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.5.0 || ^0.6.0;
+
+import "../pool/VariableYieldMath.sol";
+
+/**
+ * Wrapper for the Variable Yield Math Smart Contract Library, with return values for reverts.
+ */
+contract VariableYieldMathWrapper {
```

Comment:
Instead of Variable, I would suggest just using "cDai" or "cToken". It's true that it's more general than that, but "variable" is too generic. "cTokenYieldMath" connects cTokens to the Yield Math idea, which may be as close as we can get.

Reply:
The Pool would definitely be a `cPool`, since it connects to Compound.

The library, on the other hand, would work for any token that has a variable exchange rate towards Dai, not only cDai. We can use it for Chai, obviously, and any others.

I'm not too attached to "Variable Yield" (although vyDai/fyDai has a thing to it), but I'm not sure we should use cDai in the library either.